lib/takeaway.rb

```diff
@@ -0,0 +1,56 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'text'
+require_relative 'menu'
+
+require 'rubygems'
+require 'twilio-ruby'
+
+class Takeaway
+
+include Text, Order, Menu
+attr_accessor :menu_choice, :quantity_total, :total_count
+
+  def initialize
+    @menu_choice = {}
+    @quantity_total = []
+    @total_count = 0
+  end
+
+  def select_item
+    while @item != "finish" do
+      puts "Please type in your order or 'finish' to complete your order."
```

Comment:
don't build a command-line interface into your objects - that is a separate responsibility.   Your `Takeaway` object should be responsible for receiving orders and confirming them.  Nothing else.